wp-content/themes/oceanwp-child/functions.php

```diff
@@ -259,4 +260,70 @@ function add_related_news_to_initiative_pages() {
          
     }
  
+}
+
+
+add_action( 'wp_ajax_follow_post', 'follow_initiative' );
+function follow_initiative() {
+	global $wpdb; 
+    
+        
+	if ( !is_user_logged_in() ) {
+
+		wp_redirect('http://localhost/ggrc_website/'); 
+		
+		exit;
+	}
+	else{
+	
+		$current_user_id = get_current_user_id();
+		$postid = $_POST['postid'];
+
+		//echo $current_user_id;
```

Comment:
delete comment.
Could you also please reduce the amount of newlines?

Reply:
Deleted